pages/blog/index.vue

```diff
@@ -97,18 +97,31 @@ export default {
     const resolve = require.context('~/content/blog/', true, /\.md$/)
     const imports = resolve.keys().map((key) => resolve(key))
 
+    const tags = {
+      tokens: 'Tokens',
+    }
+
     const latestPosts = imports.sort(
       (a, b) => +new Date(b.attributes.date) - +new Date(a.attributes.date)
     )
 
-    const tokensPosts = latestPosts.filter(
-      (post) => post.attributes.tags === 'Tokens'
+    return latestPosts.reduce(
+      (acc, obj, index) => {
+        if (index === 0) {
+          acc.featured.push(obj)
+        } else if (obj.attributes.tags === tags.tokens) {
+          acc.tokensPosts.push(obj)
+        } else {
+          acc.posts.push(obj)
+        }
+        return acc
+      },
+      {
+        featured: [],
+        posts: [],
+        tokensPosts: [],
+      }
     )
```

Comment:
1. change `obj` to `post`
2. change from nested `else if` to early return, makes for a easier read

```suggestion
return latestPosts.reduce(
      (acc, post, index) => {
        if (index === 0) {
          acc.featured.push(post)
          return post
        }  
        if (post.attributes.tags === tags.tokens) {
          acc.tokensPosts.push(post)
          return acc
        } 
          acc.posts.push(obj)
        return acc
      },
      {
        featured: [],
        posts: [],
        tokensPosts: [],
      }
    )
    ```

Reply:
Thanks will look at the suggestions today and make updates 👍